data_management/tests/test_api.py

```diff
@@ -393,7 +393,7 @@ def test_filter_by_title(self):
         self.assertEqual(response['Content-Type'], 'application/json')
         results = response.json()['results']
         self.assertEqual(len(results), 1)
-        self.assertEqual(results[0]['doi_or_unique_name'], '10.15586/jptcp.v27iSP1.691')
+        self.assertEqual(results[0]['other_unique_name'], '10.15586/jptcp.v27iSP1.691')
```

Comment:
Again, I think this is an identifier:
```suggestion
        self.assertEqual(results[0]['identifier`], 'https://dx.doi.org/10.15586%2Fjptcp.v27iSP1.691')
```